docs/tutorial/tutorial.md

```diff
@@ -192,14 +220,33 @@ Instead, the best solution here is to store this state in the Board component in
 
 Pulling state upwards like this is common when refactoring React components, so let's take this opportunity to try it out. Add an initial state for Board containing an array with 9 nulls, corresponding to the 9 squares:
 
-```javascript
+```javascript{4-6}
 class Board extends React.Component {
   constructor() {
     super();
     this.state = {
       squares: Array(9).fill(null),
     };
   }
+  renderSquare(i) {
+    return <Square />;
+  }
+  render() {
+    return (
+      <div>
+        <div className="status">{status}</div>
+        <div className="board-row">
+          {this.renderSquare(0)}{this.renderSquare(1)}{this.renderSquare(2)}
```

Comment:
Also I think this might give people the wrong impression that JSX requires you put things one one line or they’ll be split by newlines. This is not true. So maybe wordier version is clearer in this case.